conf/modules/allele_specific_calling.config

```
@@ -41,11 +46,11 @@ process {
 
     withName: '.*ALLELE_SPECIFIC_CALLING:SPLIT_MULTIALLELICS' {
         ext.prefix = { "${meta.id}_norm" }
-        ext.args = '--output-type z --multiallelics -both'
+        ext.args = '--output-type z --multiallelics -both --write-index=tbi'
```

```suggestion
        ext.args = { '--output-type z --multiallelics -both --write-index=tbi' }
```
Just for consistency with he rest of the code :)